### DISCUSSION TITLE
How to be immedietally alerted when SO detects a specific alert

### DISCUSSION THREAD

**Discussion by thegreekman** [Security-Onion-Solutions/securityonion#3897] [Unsupported Versions] [answered]
Hello. I was wondering if it was possible for Security Onion to prioritize alerts in the sense that, if a specific windows event is detected, it could send out an email to a specific user or something of the sorts..

**Comment by GuardianCosmos** [comment, marked as answer]
Yes,  alerts are driven through elastalert, so you can write custom alerting as talked about here: https://docs.securityonion.net/en/2.3/elastalert.html
and also that will reference the full elastalert docs https://elastalert.readthedocs.io/en/latest/ where you can find how to customize the alerts further and specify which alerts you want to fire
By default, it will send the alert for all alerts if you leave it as is and just turn it on, but you can write specific alert rules for a given alert type or pattern match of a set of alerts

**Reply by weslambert** [reply]
Additionally, consider looking at Playbook to make it a bit easier: https://docs.securityonion.net/en/2.3/playbook.html
